plugin/ignite-plugin/src/main/scala/org/finos/vuu/table/virtualized/RollingCache.scala

```diff
@@ -0,0 +1,18 @@
+package org.finos.vuu.table.virtualized
+
+import org.finos.vuu.core.table.RowData
+import org.finos.vuu.table.virtualized.cache.GuavaRollingRowDataCache
+
+trait RollingCache[KEY, VALUE] {
```

Comment:
Wondering if 'RollingCache' is the most fortunate name for this trait ? The contract does not suggest anything 'rolling' here ? i.e. it can but does it have to ?

Reply:
Agreed, renamed from Rolling to Windowed